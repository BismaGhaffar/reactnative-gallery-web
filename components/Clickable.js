import styled from 'styled-components'

const Clickable = styled.a`
  cursor: pointer;
  text-decoration: none;
`

export default Clickable
