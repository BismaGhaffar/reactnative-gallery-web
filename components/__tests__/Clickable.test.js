import React from 'react'
import renderer from 'react-test-renderer'
import Clickable from '../Clickable'

it('Clickable can be created', () => {
  const comp = renderer.create(<Clickable />)
  expect(comp).toBeDefined()
})

it('<Clickable /> toMatchSnapshot', () => {
  const tree = renderer.create(<Clickable />).toJSON()
  expect(tree).toMatchSnapshot()
})
