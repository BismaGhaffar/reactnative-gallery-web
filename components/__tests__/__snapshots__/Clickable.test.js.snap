// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Clickable /> toMatchSnapshot 1`] = `
<a
  className="Clickable-s3in9z9-0 fGsmJS"
/>
`;
